_sections/COVID-19/en/05-this-is-not-normal-flu.md

```diff
@@ -1,25 +1,4 @@
-If this strain of coronavirus is like other viruses, aggressive measures that reduce transmission may also lower the average ‘dose’ of viral particles that cause any given case; this might reduce the average severity of disease and decrease overall death rates. Time will tell.
-
-Moreover, even IF the true COVID-19 mortality rate is as low as 1% ([as it is so far with the Diamond Princess](https://wwwnc.cdc.gov/eid/article/26/6/20-0452_article)), that would already be [ten times higher than the mortality rate of a typical seasonal flu](https://www.bloomberg.com/opinion/articles/2020-03-05/how-bad-is-the-coronavirus-let-s-compare-with-sars-ebola-flu).
-
-![Bar chart showing COVID-19 mortality rate estimated between 0.5% and 4.0%, versus Seasonal Flu at 0.1%](/images/mortality-rate.svg)
-
-> The 61,099 flu-related deaths in the U.S. during the severe flu season of 2017-2018 amounted to 0.14% of the estimated 44.8 million cases of influenza-like illness. 
-There were also an estimated 808,129 flu-related hospitalizations, for a rate of 1.8%. 
-Assume a COVID-19 outbreak of similar size in the U.S., multiply the death and hospitalization estimates by five or 10, and you get some really scary numbers: 300,000 to 600,000 deaths, and [4 million to 8 million hospitalizations in a country that has 924,107 staffed hospital beds](https://www.bloomberg.com/opinion/articles/2020-03-05/how-bad-is-the-coronavirus-let-s-compare-with-sars-ebola-flu) \[for all illnesses combined\].
-
-For most people, infection results in a mild but still transmissible illness; **that is how it spreads.** Those that get severe illness are hit really hard. Mortality is not the full picture: Italy reports that [10% of cases need not just hospitalization but also ICU care](https://twitter.com/marcelsalathe/status/1235662457261023232) -- and they need that care over a period of [3-6 weeks](https://www.washingtonpost.com/health/2020/03/07/how-doctors-treat-sickest-coronavirus-patients/). This is unsustainable.
+In pandemics, it can take years to determine the infection fatality rate. This is normal. Most experts now estimate the range to be between [.5% and 1%](https://twitter.com/CT_Bergstrom/status/1267311325765500930). However, keep in mind that even 1% would be ten fold higher than a typical seasonal influenza year. Moreover, mortality is not the full picture. Coronavirus can [affect multiple organs](https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30558-4/fulltext). Even for those who recover, there could be longer-term implications that are not yet well understood.
```

Comment:
I would drop _year_ for the sentence to read: _...it would be ten fold higher than a typical seasonal influenza._

Reply:
"Ten fold" -> ten times

Reply:
Should [MIS-C](https://www.cdc.gov/mis-c/hcp/) be mentioned, here or somewhere?

Reply:
Yes, I will add an issue and do a new pr for that.